make.sh

```diff
@@ -1,5 +1,5 @@
-export ODA_SITE=${ODA_SITE:-}
-export ODA_NAMESPACE=${ODA_NAMESPACE:-staging-1-3}
+export ODA_ENVIRONMENT=${ODA_ENVIRONMENT:-}
```

Comment:
I kind of move to using ODA_SITE. It's more explicit about referring to place of deployment. Do you need it to be environment?